library/std/src/os/unix/net/addr.rs

```diff
@@ -154,13 +153,12 @@ impl SocketAddr {
     /// Creating a `SocketAddr` with a NULL byte results in an error.
     ///
     /// ```
-    /// #![feature(unix_socket_creation)]
     /// use std::os::unix::net::SocketAddr;
     ///
-    /// assert!(SocketAddr::from_path("/path/with/\0/bytes").is_err());
+    /// assert!(SocketAddr::from_pathname("/path/with/\0/bytes").is_err());
     /// ```
-    #[unstable(feature = "unix_socket_creation", issue = "93423")]
-    pub fn from_path<P>(path: P) -> io::Result<SocketAddr>
+    #[stable(feature = "unix_socket_creation", since = "1.60.0")]
```

Comment:
Latest is 1.61

```suggestion
    #[stable(feature = "unix_socket_creation", since = "1.61.0")]
```

Reply:
Also discussed here: https://rust-lang.zulipchat.com/#narrow/stream/219381-t-libs/topic/Stabilization.20of.20unix_socket_creation.

If possible it would be great if this could be backported as it's blocking Mio v1.